e2e/tools/validator/src/validator/prometheus/__init__.py

```diff
@@ -125,6 +129,20 @@ def mape(actual: npt.ArrayLike, expected: npt.ArrayLike) -> ValueOrError:
         return ValueOrError(value=0, error=str(e))
 
 
+def mean_difference(actual: npt.ArrayLike, expected: npt.ArrayLike) -> ValueOrError:
+    try:
+        return ValueOrError(value=100 * np.abs(np.mean(actual) - np.mean(expected)) / np.mean(actual))
+    except Exception as e:
+        return ValueOrError(value=0, error=str(e))
+
+
+def std_difference(actual: npt.ArrayLike, expected: npt.ArrayLike) -> ValueOrError:
+    try:
+        return ValueOrError(value=100 * np.abs(np.std(actual) - np.std(expected)) / np.std(actual))
```

Comment:
This looks like its the difference in standard deviations between actual and expected.

Reply:
any suggestions?

Reply:
could you please shed some light into why we are comparing the difference in standard deviation  and expressing it as a ratio 🤔 ?

Reply:
we can do the F test using this ratio to check if the two series are consistent.

Reply:
@sthaha @KaiyiLiu1234 you are right, i just read F test again and found i should use the variance ratio, instead of the ratio diff. Will change it.